qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -146,7 +146,11 @@
                     image_snapshot = yes
                     mem = 32768
                     size_mem1 = 32G
```

Comment:
pre_command = 'mount -o remount,size=${size_mem1} /dev/shm'
I think adding here is more in line with expectations

Reply:
I overlooked the issue, ignore this suggestion, please